app/views/user/index.html.erb

```diff
@@ -1 +1,5 @@
-<h1>index user</h1>
+<% @users.each do | user|%>
+ <div class="border border-dark bg-white">
+  <%= render partial:'partials/user', locals: { user: user} %>
```

Comment:
Please create a link to the user's show page on `app/views/users/index.html.erb`so that the user is able to click and go to that specific user's show page.
![req](https://user-images.githubusercontent.com/35504149/151188299-6921a748-8954-44dd-85c5-347aa0a71580.png)